docs/source/en/modular_diffusers/getting_started.md

```diff
@@ -0,0 +1,1393 @@
+        "VaeImageProcessor"
+      ]
+    }
+  ],
+```
+Unlike standard repositories where components must be in subfolders within the same repo, modular repositories can fetch components from different repositories based on the `loading_specs` dictionary. e.g. the `text_encoder` component will be fetched from the "text_encoder" folder in `stabilityai/stable-diffusion-xl-base-1.0` while other components come from different repositories.
+
+
+### Creating a `ModularPipeline` from `ModularPipelineBlocks`
+
+Each `ModularPipelineBlocks` has an `init_pipeline` method that can initialize a `ModularPipeline` object based on its component and configuration specifications.
+
+Let's convert our `t2i_blocks` (which we created earlier) into a runnable `ModularPipeline`:
+
+```py
+# We already have this from earlier
+t2i_blocks = SequentialPipelineBlocks.from_blocks_dict(TEXT2IMAGE_BLOCKS)
+
+# Now convert it to a ModularPipeline
+modular_repo_id = "YiYiXu/modular-loader-t2i"
+t2i_pipeline = t2i_blocks.init_pipeline(modular_repo_id)
+```
+
+<Tip>
+
+💡 We recommend using `ModularPipeline` with Component Manager by passing a `components_manager`:
+
+```py
+>>> components = ComponentsManager()
+>>> pipeline = blocks.init_pipeline(modular_repo_id, components_manager=components)
+```
+
+This helps you to:
+1. Detect and manage duplicated models (warns when trying to register an existing model)
+2. Easily reuse components across different pipelines
+3. Apply offloading strategies across multiple pipelines
+
+You can read more about Components Manager [here](TODO)
+
+</Tip>
+
+
+### Creating a `ModularPipeline` with `from_pretrained`
+
+You can create a `ModularPipeline` from a HuggingFace Hub repository with `from_pretrained` method, as long as it's a modular repo:
+
+```py
+# YiYi TODO: this is not yet supported actually 😢, need to add support
+from diffusers import ModularPipeline
+pipeline = ModularPipeline.from_pretrained(repo_id, components_manager=..., collection=...)
+```
+
+Loading custom code is also supported:
+
+```py
+from diffusers import ModularPipeline
+modular_repo_id = "YiYiXu/modular-diffdiff"
+diffdiff_pipeline = ModularPipeline.from_pretrained(modular_repo_id, trust_remote_code=True)
+```
+
+### Loading components into a `ModularPipeline`
+
+Unlike `DiffusionPipeline`, when you create a `ModularPipeline` instance (whether using `from_pretrained` or converting from pipeline blocks), its components aren't loaded automatically. You need to explicitly load model components using `load_components`:
+
+```py
+# This will load ALL the expected components into pipeline
+import torch
+t2i_pipeline.load_default_components(torch_dtype=torch.float16)
+t2i_pipeline.to("cuda")
+```
+
+All expected components are now loaded into the pipeline. You can also partially load specific components using the `names` argument. For example, to only load unet and vae:
+
+```py
+>>> t2i_pipeline.load_components(names=["unet", "vae"], torch_dtype=torch.float16)
```

Comment:
Maybe not needed but what happens if I try to pass a name to `names` that's not supposed to be passed? Maybe we could show an example of the error?